apps/zotonic_mod_admin/src/support/z_admin_rsc_import.erl

```diff
@@ -26,16 +26,19 @@
 -include("zotonic_core/include/zotonic.hrl").
 
 event(#postback{ message={import_refresh, Args} }, Context) ->
+    OnError = proplists:get_value(on_error, Args),
     {id, Id} = proplists:lookup(id, Args),
     case m_rsc_import:reimport_recursive_async(Id, Context) of
         {ok, {_Id, _ObjectIds}} ->
             case proplists:get_all_values(on_success, Args) of
                 [] ->
-                    z_render:growl(?__("Succesfully imported page from the remote server.", Context), Context);
+                    Context1 = z_render:wire(OnError, Context),
+                    z_render:growl(?__("Succesfully imported page from the remote server.", Context1), Context1);
```

Comment:
Should it be an error or success in this case - the onerror seems to contradict the growl.

Reply:
Ooop, that is indeed an ok!